app.py

```diff
@@ -185,17 +186,39 @@ def set_frame_range(self, engine, in_frame, out_frame):
 
         elif engine == "tk-nuke":
             import nuke
-
+            root = nuke.root()
+            lock_range = root["lock_range"]
             # unlock
-            locked = nuke.root()["lock_range"].value()
+            locked = lock_range.value()
+            change_msg = 'from %s-%s to %s-%s' % (prev_in, prev_out, in_frame, out_frame)
             if locked:
-                nuke.root()["lock_range"].setValue(False)
+                box = QtGui.QMessageBox()
+                box.setText("Frame range is currently locked,\n"
+                            "Do you want to still update it %s ?" % change_msg)
```

Comment:
`Do you still want to update it` from what people say in the developpers channel in Slack.